landing-page/index.html

```diff
@@ -0,0 +1,33 @@
+<html>
+  <head>
+    <style>
+      body {
+  background-color: MistyRose;
```

Comment:
This style rule should be indented underneath the body selector by 2 spaces (I think you're using 2 spaces? Whether 2 or 4, or if you're using tabs, just keep it consistent).